.circleci/config.yml

```diff
@@ -15,14 +15,10 @@ jobs:
       - run:
           name: Run iOS tests
           command: make test-ios
-          environment:
-            SNAPSHOT_ARTIFACTS: /tmp/__SnapshotArtifacts__
```

Comment:
This wasn't working with Circle as expected for `xcodebuild`. Had to add to the schemes via XcodeGen.